Student/Student/RouterViewController.swift

```diff
@@ -29,12 +29,18 @@ struct DebugRoute {
 
 let routes = [
     DebugRoute("/login"),
-    DebugRoute("/courses"),
-    DebugRoute("/courses/177/assignments/1933"),
-    DebugRoute("/courses/167/quizzes"),
-    DebugRoute("/courses/162/assignments/1916/fileupload"),
-    DebugRoute("courses/162/assignments/1901/submissions/12", [.modal, .embedInNav]),
-    DebugRoute("courses/162/assignments/1901/submissions/12/urlsubmission", [.modal, .embedInNav]),
+//    DebugRoute("/courses"),
+//    DebugRoute("/courses/177/assignments/1933"),
+//    DebugRoute("/courses/167/quizzes"),
+//    DebugRoute("/courses/162/assignments/1916/fileupload"),
+//    DebugRoute("courses/162/assignments/1901/submissions/12", [.modal, .embedInNav]),
+//    DebugRoute("courses/162/assignments/1901/submissions/12/urlsubmission", [.modal, .embedInNav]),
```

Comment:
Should we remove these? Or was it not intentional to commit the change?

Reply:
Ah, yes, i'll remove